Formula/kubernetes-service-catalog-client.rb

```diff
@@ -0,0 +1,33 @@
+class KubernetesServiceCatalogClient < Formula
+  desc "Consume Services in k8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
+      :revision => "87a5db0e1e0359ce372037a235ce4448944c8611"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["NO_DOCKER"] = "1"
+    dir = buildpath/"src/github.com/kubernetes-incubator/service-catalog"
+
+    dir.install buildpath.children
+
+    cd dir do
+      ldflags = %W[
+        -s -w -X
+        github.com/kubernetes-incubator/service-catalog/pkg.VERSION=v#{version}
+      ]
+      system "go", "build", "-ldflags", ldflags.join(" "), "-o",
+             "bin/svcat", "./cmd/svcat"
```

Comment:
If you change this to `bin/"svcat"` you should be able to get rid of the `bin.install "bin/svcat"` below?

Reply:
I did not know that. Thanks for letting me know. I have updated it.